src/libexec/zmfixperms

```diff
@@ -265,6 +265,18 @@ if [ -d /opt/zextras ]; then
     fi
   fi
 
+  if [ -d /opt/zextras/docs ]; then
+    printMsg "Fixing permissions and ownership on /opt/zextras/docs"
+    if [ -d /opt/zextras/docs/conf ]; then
```

Comment:
Are we sure that this dir `/opt/zextras/docs/conf` has the correct owner?

Reply:
I checked the pkgbuild seemes like they put some backup files before installation
I wanted it to be a variable in case they decided to rename this guy but this probably never gonna happen so I will do whatever you think its better bcz i dont know (yagni)

Reply:
> yagni
TIL